Esterni/ManualeUtente/components/c1-Introduzione.tex

```diff
@@ -0,0 +1,13 @@
+\section{Introduzione}
+\subsection{Scopo del documento}
+Lo scopo di questo documento è illustrare tutte le funzionalità del \gloman{sync engine} richiesto dal \textit{capitolato C7}. L'utente finale in questo modo avrà dunque a disposizione tutte le indicazioni per il corretto uso del \gloman{software}.
+
+\subsection{Scopo del prodotto}
+Lo scopo del capitolato \textit{C7} è la creazione di un algoritmo \gloman{solido} ed efficiente in grado di garantire il salvataggio e la sincronizzazione dei cambiamenti presenti in \gloman{cloud}. È inoltre richiesta la creazione di una interfaccia grafica multipiattaforma. Il progetto deve dunque funzionare sui principali sistemi operativi desktop quali Windows, macOS e Linux senza richiedere l'installazione manuale di ulteriori prodotti per il corretto funzionamento. 
```

Comment:
cambiere "il salvataggio e la sincronizzazione dei cambiamenti presenti in \gloman{cloud}" con

il salvataggio e la sincronizzazione di file attraverso il \gloman{cloud}